docs/docs/config/log-level.md

```diff
@@ -3,11 +3,12 @@ title: 'log-level'
 language: 'en'
 ---
 
-This property enables log level filter. Default is "OFF".
+This property enables log level filter and file. The default level is "OFF" and the logs are not logged to a file as default.
 
 ```toml
 [developer]
 log-level = "OFF"
+log-file = "rio.log"
```

Comment:
I think it should a boolean tbh, the name of the file should not change unless used another configuration property. What do you think?

Reply:
Done
cd4c280add17913b34ed78e42cafb9fa88943677